src/main/scala/com/gu/scanamo/scrooge/ScroogeDynamoFormat.scala

```diff
@@ -61,7 +61,7 @@ class ScroogeDynamoFormatMacro(val c: blackbox.Context) {
             val format = $formatWithFallback
             val possibleValue = collection.convert.WrapAsScala.mapAsScalaMap(av.getM).get(${termName.toString}).map(format.read).orElse(format.default.map(cats.data.Xor.right))
             val validatedValue = possibleValue.getOrElse(cats.data.Xor.left[com.gu.scanamo.error.DynamoReadError, $tpe](com.gu.scanamo.error.MissingProperty))
-            validatedValue.leftMap(e => com.gu.scanamo.error.InvalidPropertiesError(cats.data.NonEmptyList(com.gu.scanamo.error.PropertyReadError(${termName.toString}, e)))).toValidated
+            validatedValue.leftMap(e => com.gu.scanamo.error.InvalidPropertiesError(cats.data.NonEmptyList(com.gu.scanamo.error.PropertyReadError(${termName.toString}, e), Nil))).toValidated
```

Comment:
`NonEmptyList.of` is probably more idiomatic and should avoid the need for the `Nil`.

https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/NonEmptyList.scala#L127